log/log.go

```diff
@@ -265,3 +268,30 @@ func Warnw(msg string, keyvalues ...interface{}) {
 func Errorw(err error, msg string) {
 	Logger().Error().Err(err).Msg(msg)
```

Comment:
what about Errorw?

Reply:
yeah :eyes: i'll just KISS and print stack on panic.